server/helpers/has-graphics-access.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const {GRAPHIC_SYNDICATION_PRODUCT_CODE} = require('config');
+module.exports = exports = (prodList= []) => {
+	return (prodList.length >= 1) ? prodList.some(product => product.code === GRAPHIC_SYNDICATION_PRODUCT_CODE): false;
```

Comment:
Sorry for late comment, I only spotted it now:

There's no need to have a ternary operator, `some` applied to empty array will return false
```
> [].some(a => a)
false
```

Reply:
of course 🤦